packages/adblocker/src/engine/bucket/network.ts

```diff
@@ -53,6 +61,13 @@ export default class NetworkFilterBucket {
       optimize: config.enableOptimizations ? optimizeNetwork : noopOptimizeNetwork,
     });
 
+    this.htmlIndex = new ReverseIndex({
```

Comment:
should `matchAll` iterate through the htmlIndex? I guess it should, please confirm @remusao